internal/isoeditor/rhcos_test.go

```diff
@@ -68,13 +66,10 @@ var _ = Context("with test files", func() {
 	Describe("CreateMinimalISOTemplate", func() {
 		It("iso created successfully", func() {
 			editor := editorForFile(isoFile, workDir)
+			err := editor.(*rhcosEditor).embedOffsetsInSystemArea(isoFile)
+			Expect(err).ToNot(HaveOccurred())
 			file, err := editor.CreateMinimalISOTemplate(defaultTestServiceBaseURL)
 			Expect(err).ToNot(HaveOccurred())
-
-			// Creating the template should remove the working directory
-			_, err = os.Stat(workDir)
-			Expect(os.IsNotExist(err)).To(BeTrue())
```

Comment:
Is this no longer true? Should we be removing the working directory manually rather than removing this assertion?

Reply:
It was previously removed by the `Finalize` call to go-diskfs which we are no longer using after this change.

Reply:
just a wrong 'cut-paste' from the CreateClusterMinimalISO test :)